scss/prototype/_display.scss

```diff
@@ -0,0 +1,62 @@
+// Foundation for Sites by ZURB
+// foundation.zurb.com
+// Licensed under MIT Open Source
+
+////
+/// @group prototype-display
+////
+
+/// Responsive breakpoints for display classes
+/// @type Boolean
+$prototype-display-breakpoints: $global-prototype-breakpoints !default;
+
+/// Map containing all the `display` classes
+/// @type Map
+$prototype-display-classes: (
+  none, inline, inline-block, block, table, table-cell
+) !default;
+
+/// Map containing all the `display-flex` classes
+/// @type Map
+$prototype-display-flex-classes: (
+  flex, inline-flex
+) !default;
+
+@mixin foundation-prototype-display {
+  @each $display in $prototype-display-classes {
+    .display-#{$display} {
+      display: $display !important;
```

Comment:
I think it's soul'd be better if you create a mixing "foundation-display" and use it. 
```
@mixin foundation-display($display) {
    display: $display !important;
}
```

It would allow to easily be passed to the prototyping by class to sementic version.